src/lib/style/abstracts/_variables.scss

```diff
@@ -8,7 +8,7 @@ $gray-light: #f9f9f9 !default;
 $primary-action-color: rgb(64, 123, 160) !default;
 $primary-bgcolor-rgba: rgba(201, 66, 129, 0.03) !default;
 
-$secondary-action-color: #5960a5 !default;
+$secondary-action-color: rgb(98, 172, 98) !default;
```

Comment:
Changed default from purple to 'Synapse Green' so any components that inadvertently end up in SWC won't look too out of place.